content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -32,9 +42,9 @@ instead. Also, the
 imperative whereas Deployments are declarative, so we recommend using Deployments
 through the [`rollout`](/docs/reference/generated/kubectl/kubectl-commands#rollout) command.
 
-While ReplicaSets can be used independently, today it's mainly used by
-[Deployments](/docs/concepts/workloads/controllers/deployment/) as a mechanism to orchestrate pod
-creation, deletion and updates. When you use Deployments you don't have to worry
+While ReplicaSets can be used independently, it could also be used by other resources. For example,
```

Comment:
I think rather than saying ReplicaSets can be used by Deployments, we should say that creating a Deployment is the recommended way of using a ReplicaSet.

Reply:
done